charts/gh-action-exporter/values.yaml

```diff
@@ -14,10 +14,6 @@ imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
-
-config:
-  job_relabelling: []
-
```

Comment:
why?

Reply:
Ok now I understood why, we need this feature as it does more than handling costs. It's used for other purposes.